config/application.rb

```diff
@@ -14,6 +14,11 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 7.0
 
+    # Please, add to the `ignore` list any other `lib` subdirectories that do
+    # not contain `.rb` files, or that should not be reloaded or eager loaded.
+    # Common ones are `templates`, `generators`, or `middleware`, for example.
+    config.autoload_lib(ignore: %w[assets tasks])
```

Comment:
Adding warden here might work:

```suggestion
    config.autoload_lib(ignore: %w[assets tasks warden])
```

Reply:
Because I think this change means we're trying to load `lib/warden/strategies/omniauth/rb` everywhere, which I don't think we were doing before